tests/test_lime_survey.py

```diff
@@ -208,6 +209,103 @@ def test_lime_system_info_dtypes(self):
+        expected_response = [
+            'A1', 'A1', 'A3', 'A2', 'nan',
+            'A1', 'A1', 'A1', 'A1', 'A1'
+        ]
+        response = self.survey.get_responses(self.single_choice_column, labels=False)
+        np.testing.assert_array_equal(
+            expected_response,
+            response.values.flatten().astype(str)[:10]
+        )
+
+        expected_response = [
+            'Woman', 'Woman', 'Man', "I don't want to answer this question",
+            'nan', 'Woman', 'Woman', 'Woman', 'Woman', 'Woman'
+        ]
+        response = self.survey.get_responses(self.single_choice_column, labels=True)
+        np.testing.assert_array_equal(
+            expected_response,
+            response.values.flatten().astype(str)[:10]
+        )
+
+    def test_get_response_multiple_choice(self):
+        """Test get response for multiple choice question type"""
+        expected_response = [
+            False, False, False, False, False,
+            False, False, False, False, False
+        ]
+        response = self.survey.get_responses(self.multiple_choice_column, labels=False)
+        np.testing.assert_array_equal(expected_response, response.values.flatten()[:10])
+        response = self.survey.get_responses(self.multiple_choice_column, labels=True)
+        np.testing.assert_array_equal(expected_response, response.values.flatten()[:10])
+
+    def test_get_response_free(self):
+        """Test get response for free question type"""
+        expected_response = [
+            '2017-01-01 00:00:00', '2020-06-01 00:00:00',
+            '2019-08-01 00:00:00', '2017-05-01 00:00:00',
+            'nan', '2017-08-01 00:00:00',
+            '2018-01-01 00:00:00', '2020-09-01 00:00:00',
+            '2017-08-01 00:00:00', '2019-12-01 00:00:00'
+        ]
+        response = self.survey.get_responses(self.free_column, labels=False)
+        np.testing.assert_array_equal(
+            expected_response,
+            response.values.flatten().astype(str)[:10]
+        )
+        response = self.survey.get_responses(self.free_column, labels=True)
+        np.testing.assert_array_equal(
+            expected_response,
+            response.values.flatten().astype(str)[:10]
+        )
+
+    def test_get_response_array(self):
+        """Test get response for array question type"""
+        expected_response = [
+            'A1', 'A1', 'A2', 'A1', 'A1',
+            'A1', 'A1', 'A1', 'A3', 'A1'
+        ]
+        response = self.survey.get_responses(self.array_column, labels=False)
+        np.testing.assert_array_equal(expected_response, response.values.flatten()[:10])
+
+        expected_response = [
+            'Yes', 'Yes', 'No', 'Yes', 'Yes',
+            'Yes', 'Yes', 'Yes', 'I don\'t know', 'Yes'
+        ]
+        response = self.survey.get_responses(self.array_column, labels=True)
+        np.testing.assert_array_equal(expected_response, response.values.flatten()[:10])
+
```

Comment:
Column name

Reply:
I can add this but not before friday I think.